hax/hax/types.py

```diff
@@ -42,6 +42,16 @@ class Uint128Struct(c.Structure):
 ObjT.__doc__ = 'Motr conf object types and their m0_fid.f_container values'
 
 
+FidT = Enum(
```

Comment:
According to the way how FidT is used, it feels more like a static map `fid container` -> `string name`. Why wouldn't we convert it into the static dict then?

Reply:
Yeah, it was a leftover. fixed the usage.

Reply:
Sorry, I still don't understand why you need it to be a Enum. According to what I see, you need the mapping, it doesn't need to be a separate type. And you use it like a dict, actually

```python
    def fid2objT(self, fid: str) -> str:
        """
        Returns ObjT for a given fid as a string.
        """
        cont = f'0x{Fid.parse(fid).container:x}'
        return str(FidT[cont].value)
```

So effectively FidT could either be a dict or a function that performs this mapping (while dict could be an implementation detail). And I would vote for function. Do I miss anything?

Reply:
Removed, not needed. Thanks.